app/models/event_type.rb

```diff
@@ -10,6 +10,7 @@ class EventType < ActiveRecord::Base
 
   alias_attribute :name, :title
 
+  # If LENGTH_STEP > 60 the public schedule must be changed
```

Comment:
Can we somehow force that the step is between 15 and 60 minutes? Changes the schedule is not that easy!